api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -574,7 +565,7 @@ public static Module getModuleByPackage(String modulePackage) {
 	 * @see Daemon#startModule(Module)
 	 */
 	public static Module startModule(Module module) throws ModuleException {
-		return Daemon.startModule(module);
```

Comment:
I presume you are talking about the section i deleted that was taking care of missing dependencies. I did that because i moved the functionality to [here](https://github.com/ivange94/openmrs-core/blob/TRUNK-4668/api/src/main/java/org/openmrs/module/ModuleFactory.java#L604) and since ModuleFactory.startModules() makes a call to ModuleFactory.startModule(module) [here](https://github.com/ivange94/openmrs-core/blob/TRUNK-4668/api/src/main/java/org/openmrs/module/ModuleFactory.java#L237) which ends up making a call to ModuleFactory.startModule(module, false, null), i didn't think it was necessary to leave that there again.